src/platform/guiwin.cpp

```diff
@@ -1626,15 +1621,16 @@ class FileDialogImplWin32 final : public FileDialog {
     }
 
     bool RunModal() override {
-        if(GetFilename().IsEmpty()) {
-            SetFilename(Path::From(_("untitled")));
-        }
-
-        if(isSaveDialog) {
-            return GetSaveFileNameW(&ofn) == TRUE;
-        } else {
-            return GetOpenFileNameW(&ofn) == TRUE;
-        }
+	if(isSaveDialog) {
+	    SetTitle(C_("title", "Save File"));
+	    if(GetFilename().IsEmpty()) {
```

Comment:
Use spaces, not tab for white space.

Reply:
I am still new to GitHub, do I create another pull request, or just update the code and let you know?

Reply:
Push the change to your repository and it should show up here.

Better yet, fix the whitespace and squash the 2 commits, then push to your repository and it will show up here as a single commit PR.

It does take time to get used to this stuff :-)

Reply:
After squashing it would require using --force to your repo.

Reply:
Change the file.
git add guiwin.cpp
git commit --amend
got push yourremote --force